integration/pvtdata/pvtdata_test.go

```diff
@@ -498,14 +487,6 @@ var _ = Describe("PrivateData", func() {
 			}
 
 			When("chaincode is migrated from legacy to new lifecycle with same collection config", func() {
```

Comment:
Since we are keeping assertReconcileBehavior() in here but not the upgrade from legacy chaincode, change the description text in the When().

Reply:
fixed